lib/frame-manager.js

```diff
@@ -42,10 +41,15 @@ function FrameManager(window) {
     if (!self.listenerCount('data')) {
       parent.emit('log', 'unsubscribing from browser window frames')
       window.webContents.endFrameSubscription();
-      subscribed = false;
     }
   }
 
+  function cancelFrame() {
+    requestedFrame = false;
+    clearTimeout(frameRequestTimeout);
+    self.emit('data', null);
+  }
+
```

Comment:
Yeah, that’s why I gave two solutions :P   I think I agree with you that clarity is better, so I’d keep `cancelFrame()` around but rewrite it to just delegate to `receiveFrame(null)`.